src/Rings/integer-localizations.jl

```diff
@@ -0,0 +1,197 @@
+#################################################################################
+# A minimal implementation of the localization interface for the ring ℤ         #
+#################################################################################
+
+export FmpzComplementOfPrimeIdeal, FmpzPowersOfElement, FmpzComplementOfZeroIdeal
+export FmpzLocalizedRing, FmpzLocalizedRingElem
+export generator, ambient_ring, localize_at, parent, numerator, denominator
+
+
+#################################################################################
+# Multiplicatively closed sets given by powers of a specific integer            #
+#################################################################################
+
+@Markdown.doc """
+    FmpzPowersOfElement
+
+A minimal implementation of `AbsPowersOfElement{RingType, RingElemType}` 
+for integers of type `fmpz`.
+"""
+mutable struct FmpzPowersOfElement <: AbsPowersOfElement{FlintIntegerRing, fmpz}
+  d::Vector{fmpz} # a vector of admissible denominators. 
+
+  function FmpzPowersOfElement(denominators::Vector{fmpz})
+    return new(denominators)
+  end
+end
+
+### required getter functions
+ambient_ring(S::FmpzPowersOfElement) = ZZ
+
+### required functionality
+function Base.in(a::fmpz, S::FmpzPowersOfElement) 
+  # check whether ∃ c ∈ ℤ, k ∈ ℕ₀: c ⋅ a = (∏ᵢ dᵢ)ᵏ, where dᵢ are the admissible denominators in S.
+  d = prod(denominators(S))
+  g = gcd(a, d)
+  while(!(g==one(ZZ)))
+    a = divexact(a, g)
+    g = gcd(a, d)
+  end
+  return a == one(ZZ)
+end
+
+### additional constructors
+FmpzPowersOfElement(d::fmpz) = FmpzPowersOfElement([d])
+FmpzPowersOfElement(d::Int) = FmpzPowersOfElement(ZZ(d))
```

Comment:
Could handle arbitrary integers, too:
```suggestion
FmpzPowersOfElement(d::Integer) = FmpzPowersOfElement(ZZ(d))
```